packages/lens/documents/queries/NftGalleries.graphql

```diff
@@ -0,0 +1,21 @@
+query NftGalleries($request: NftGalleriesRequest!) {
+  nftGalleries(request: $request) {
+    id
+    name
+    profileId
+    items {
+      name
+      description
+      collectionName
+      contractAddress
+      tokenId
+      chainId
+      originalContent {
+        uri
+        animatedUrl
+      }
+    }
+    createdAt
+    updatedAt
+  }
```

Comment:
are we using all fields?

Reply:
yes except timestamps use them, same as NftFeed.

removing timestamps